admin_manual/maintenance/migrating.rst

```diff
@@ -29,3 +29,95 @@ To start, let us be specific about the use case. A configured ownCloud instance
 
 #.  Change the ``CNAME`` entry in the DNS to point your users to the new
     location.
+
+
+
+
+Example
+
+Note: For this example to work, you need ssh on both servers and the PermitRootLogin to be set to "yes" on the new server.
+
+Install SSH
+
+apt install openssh-server openssh-client -y
```

Comment:
Can you do the changes yourself? I have no Idea how to work with restructured Text, sorry